aws_lambda_powertools/utilities/batch/base.py

```diff
@@ -95,7 +95,7 @@ def failure_handler(self, record: Any, exception: Exception):
             "fail", exceptions args, original record
         """
         entry = ("fail", exception.args, record)
-        logger.debug("Record processing exception: ", exception)
+        logger.debug("Record processing exception: %s", exception)
```

Comment:
Could you change to f-string for consistency, please?

Reply:
@heitorlessa  sure.

Reply:
@heitorlessa  done.